handbook/engineering/engineering-management.md

```diff
@@ -54,3 +54,21 @@ Here are some books that multiple people on the team have read and recommend. If
   - Session 3 week of June 21
   - Session 4 week of July 5
   - Session 5 week of July 19
+
+## Temporary Management Positions
```

Comment:
I think it would be good to call out what this isn't here, something like:

```
Team lead vs Temp Manager - 
A temp manager's first priority is the team care and feeding.  This includes 1:1, various HR tasks (reviews, hiring screens, etc), outward communication, and process.  They may find they have some time to do technical work (coding, code reviews, etc), but that is secondary.

A team lead's first priority is making sure they help others get work done....etc etc
```
Needs better wordsmithing, but something like that I think will be useful